test/vm_tests/builder/codegen_test.dart

```diff
@@ -50,14 +50,16 @@ main() {
       final errorCollector = ErrorCollector.log(sourceFile, logger);
 
       declarations = parseAndValidateDeclarations(unit, errorCollector);
-      implGenerator = ImplGenerator(logger, sourceFile);
+      // FIXME add tests for both cases
+      implGenerator = ImplGenerator(logger, sourceFile, nullSafety: false);
```

Comment:
Addressing in this PR or a ticketed follow-up?

Reply:
Good catch, follow up ticket; addressing here: c5335296935ee11b5ff753e833a8fed81fb0b6f0